src/flinkStatementResultsManager.ts

```diff
@@ -119,6 +126,7 @@ export class FlinkStatementResultsManager {
     private readonly resourceLoader: CCloudResourceLoader = CCloudResourceLoader.getInstance(),
   ) {
     this._results = os.signal(new Map<string, any>());
+    this._rawResults = os.signal<any[]>([]);
```

Comment:
kinda weird type hint, but I guess we want to it know it's an array

Reply:
That's right yeah, actually now that I see this -- it does have a proper type, array of objects having two fields `op` and `raw`. I'll update this.

Reply:
Done